custom_components/media_player_template/media_player.py

```diff
@@ -97,7 +79,6 @@
 MEDIA_DURATION_TEMPLATE = "media_duration_template"
 CURRENT_SOUND_MODE_TEMPLATE = "current_sound_mode_template"
 CONF_SOUND_MODES = "sound_modes"
-CONF_UNIQUE_ID = "unique_id"
```

Comment:
I removed this and replaced it with an import (added line 29, see https://github.com/golles/media_player.template/blob/fix-for-%2347/custom_components/media_player_template/media_player.py#L29)